src/AddTodoForm.jsx

```diff
@@ -1,13 +1,28 @@
-function AddTodoForm () {
+function AddTodoForm(props) {
+
+    function handleAddTodo(event) {
+        event.preventDefault();
+
+        const todoTitle = event.target.title.value;
+
+        console.log(todoTitle);
+
+        props.onAddTodo(todoTitle);
+
+        event.target.reset();
+
+
```

Comment:
Would look nicer if you collapsed some of the spacing.  Use newlines for readability.